### ISSUE TITLE
Menu submenu overlay and speed need adjustment.

### ISSUE THREAD

**Issue by TanukiKabuki** [LadybirdBrowser/ladybird#420]
Menu rendering on Nobara Linux 39 is exceedingly fast and the overlay settings cover root menu options with the sub menu selections

**Comment by trflynn89** [comment]
Please attach screenshots that show the issue

**Comment by circl-lastname** [comment]
Which menu? The hamburger menu of the browser?

**Comment by TanukiKabuki** [comment]
Yes, the hamburger menu in the browser and the submenus within it.

**Comment by shannonbooth** [comment]
Closing, UI has had a lot of work put on it since
